Verbatim `util/sum.v` section of patches/prosa-v06-rocq93-official-proof-closure-common.patch
(sha256 6819bfa9ac0bb9cfd6bfc7182d57e22463cf7a57b38ce9af89c8e7de96868050), split out for classic official closures.
--- a/util/sum.v
+++ b/util/sum.v
@@ -3,6 +3,7 @@
 Require Export prosa.util.notation.
 Require Export prosa.util.rel.
 Require Export prosa.util.nat.
+Set SsrOldRewriteGoalsOrder.
 
 Section SumsOverSequences.
 
